starknet-replay/src/histogram/mod.rs

```diff
@@ -0,0 +1,281 @@
+//! This module is responsible for rendering and saving the SVG image of the
+//! libfunc histogram.
+//!
+//! It uses the `plotter` library to generate the histogram
+//! and rendering.
+//!
+//! SVG has been chosen because, as a vector graphics format, it makes reading
+//! text easy and zooming doesn't degrade the quality.
+//!
+//! The file `mod.rs` contains the public interface. The main way to use this
+//! module is by calling the function `export` to render and save the SVG image.
+//! The file `plot.rs` contains the call to the `plotters` library.
+
+use std::ops::{Add, Div};
+use std::path::PathBuf;
+
+use super::runner::replay_statistics::ReplayStatistics;
+use crate::error::HistogramError;
+use crate::histogram::plot::render;
+
+mod plot;
+
+/// This alias improves readability of the histogram parameters.
+type Pixel = u32;
+
+/// This struct contains the variable configuration parameters for rendering the
+/// histogram image.
+///
+/// Configuration parameters that don't depend on `ReplayStatistics` are
+/// hardcoded.
```

Comment:
If they're hardcoded then this isn't something the consumer of the API needs to know about!